pkg/restic/backupper.go

```diff
@@ -81,11 +81,6 @@ func (b *backupper) BackupPodVolumes(backup *arkv1api.Backup, pod *corev1api.Pod
 		return nil, nil
 	}
 
-	// ensure a repo exists for the pod's namespace
-	if err := b.repoManager.ensureRepo(pod.Namespace); err != nil {
```

Comment:
The item backupper won't call this function unless it knows the repo for this ns is initialized, right?

Reply:
It does not currently do that. The effect of removing this call is that error handling moves into the pod volume backup controller, since its restic call will fail (though I'll add code there to explicitly check for a `Ready` repo before attempting operations). I think that's reasonable, as the PVB will fail and include an error. WDYT?

Reply:
I'd rather not create a PVB if we don't have to